.github/workflows/deploy-asset-metadata.yml

```diff
@@ -0,0 +1,30 @@
+---
+name: Deploy Asset Metadata to App Engine
+
+on:
```

Comment:
Let's add path filter - as we don't really need to re-deploy each time master is pushed, as asset-metadata is a separate app.

https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore

Reply:
Good shout!